user_guide/src/missing_data.md

```diff
@@ -0,0 +1,171 @@
+# Missing data
+
+This page sets out how missing data is represented in `Polars` and how missing data can be filled.
+
+## `null` and `NaN` values
+
+Each `Series` or column in a `DataFrame` is an Arrow array based on the Apache Arrow format.
```

Comment:
Can a collection of `Arrow arrays`. It can be one or multiple.

The missing values in arrow data are bit encoded, having only `length / 8` bytes overhead per array.